src/core/Response.cpp

```diff
@@ -4,24 +4,22 @@
 
 typeMap Response::fileTypes_;
 
-Response::Response() {
```

Comment:
Do we ever need a public default constructor for a response? or is this more for testing

Reply:
It's just in case. The default constructor creates content that wouldn't cause any harm